quipucords/api/source/tests_source.py

```diff
@@ -118,6 +121,50 @@ def create_expect_400_with_query(self,
             response_json = response.json()
             self.assertEqual(response_json, expected_response)
 
+    #################################################
+    # Function Tests
+    #################################################
+    def test_format_source(self):
+        """Test the format source method."""
+        start = datetime.now()
+        source = Source(
+            name='source1',
+            source_type='network',
+            port=22)
+        source.save()
+        end = datetime.now()
+        scan_task = ScanTask(source=source,
```

Comment:
You should use https://github.com/quipucords/quipucords/blob/master/quipucords/scanner/test_util.py#L23 to create a test job/task.  There are a lot of dependencies now in the code for state.

Reply:
updated